src/pixelator/pixeldataset/datastores.py

```diff
@@ -590,13 +608,18 @@ def write_dataframe(
                     partitioning_flavor="hive",
                     partitioning=partitioning,
                     use_threads=False,
+                    file_options=file_options,
                     existing_data_behavior="overwrite_or_ignore",
                 )
             return
 
         self._check_if_writeable(key)
         pq.write_table(
-            pa.Table.from_pandas(dataframe), where=key, filesystem=self._file_system
+            pa.Table.from_pandas(dataframe, preserve_index=False),
```

Comment:
I think we were talking about this, but can you remind me why `presenrve_index=False` was key here?

Reply:
`preserve_index=False` means that we don't create an additional column for the index from what ever is in the pandas index. See here: https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.from_pandas 

Adding this in this context means that we preserve the behavior we have had in pixelator so far, i.e. no specific index column is stored in the files.